### PR TITLE
Wire JoinTeam in wallet

### LINKED ISSUE #10125: Join team in not wired in wallet
The wiring on the JoinTeam command is missing in the wallet.

### LINKED ISSUE #10127: ApplyReferralCode and JoinTeam shouldn't be mixed
They should not have the same verification path.

### PR DESCRIPTION
**Pull request by ValentinTrinque**
Close #10125 Close #10127

### CODE REVIEW

File: commands/join_team.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package commands
```
Comment (EVODelavega) [inline commands/join_team.go:1]: Missing the copyright blurb

Comment (ValentinTrinque) [reply commands/join_team.go:1]: Alright, time for a linter

**Comment by ze97286** [comment]
but it's not wired in transaction.go is it? I.e. nothing is calling the checkJoinTeam

**Comment by ValentinTrinque** [comment]
Good catch. I did wire everything during my test, but I may have messed up my rebase.

**Review by ze97286 — APPROVED** [review]

**Pull request merged**